README.md

```diff
@@ -226,5 +179,14 @@ Report plugin enable user report tracing/metric data to different kind of backen
 
 Refer to [Report Plugin Development Guide](./doc/report-development-guide.md)
 
+## Community
+[Github issues](https://github.com/megaease/easeagent/issues)  
+
+If you have any questions, welcome to discuss in our community. welcome to join!
+
+|              Slack Channel               |
+|:----------------------------------------:|
+| ![Slack](doc/images/slack-community.jpg) |
+
```

Comment:
Let's just use the link instead of QR Code.

Reply:
> Let's just use the link instead of QR Code.

done